src/python/pants/backend/pants_info/list_target_types.py

```diff
@@ -192,7 +192,11 @@ def format_for_cli(self, console: Console) -> str:
         output.extend(
             [
                 "Valid fields:\n",
-                *sorted(f"{field.format_for_cli(console)}\n" for field in self.fields),
+                *sorted(
+                    f"{field.format_for_cli(console)}\n"
+                    for field in self.fields
+                    if not field.alias.startswith("_")
```

Comment:
This filters out "private" fields from `./pants target-types`. We used to have this, and I think it got lost when dropping v1.